src/scss/layouts/_footer.scss

```
@@ -0,0 +1,76 @@
+.footer {
+  width: 100%;
+  background-color: $whiteColor;
+  color: $whiteColor;
+  padding-top: 18.9rem;
+  position: relative;
+
+  @include dFlexCol;
+
+  &__oursponsors {
+    background-color: $blueColor;
+    padding: 7.2rem;
+
+    @include dFlexCol($gap: 6.4rem);
+    @include positionCenter($x: 0, $y: 8rem);
+  }
+
+  &__oursponsors-heading {
+    @include heading2($color: $whiteColor);
+  }
+
+  &__oursponsors-logos {
+    @include dFlex($gap: 8.4rem);
+  }
+
+  &__content {
+    width: 100%;
+    background-color: $blackColor;
+    padding: 19.2rem 8rem 9.6rem;
+
+    @include dFlex($align: start, $gap: 24.2rem);
+  }
+  &__content-row {
+    @include dFlexCol(start, $align: start, $gap: 3.2rem);
+  }
+
+  &__content-heading {
+    @include heading4($color: $whiteColor);
+    line-height: 2;
+  }
+
+  &__content-item {
+    @include dFlexCol($gap: 1.6rem);
+
+    a {
+      width: 100%;
+
+      @include fontStyle(
```

```suggestion
      @include textBody($whiteColor);
```

Tại sao không mixin textBody